docs/knowledge_base/policy_basics.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Policy Basics
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+# Basic Policy
+
+```rego
+# This is effectively the name of the policy.
+# It must be "policy".
+package policy
+
+import data.phylum.level
+import future.keywords.contains
+import future.keywords.if
+
+issue contains "risk level cannot exceed medium" if {
+	data.issue.severity > level.MEDIUM
+}
+```
+
+This policy can be executed using `opa eval --data phylum.rego --data <YOUR POLICY>.rego --data constants.json --input input.json --schema schema --format pretty data.phylum.job`.
+
+| Input | Description | Provider |
+| --- | --- | --- |
+| `phylum.rego` | describes jobs, dependencies, and issues | Phylum |
```

Comment:
`phylum.rego` pulls together rules for jobs dependencies and issues. At least that's all it does right now. We could also add helper functions in there.

Reply:
Updated the description a bit.